src/test/java/com/PlaceLab/Amila/qamp/utils/WebDriverSetup.java

```diff
@@ -0,0 +1,27 @@
+package com.PlaceLab.Amila.qamp.utils;
```

Comment:
it is a good practice to have directory names written in lower case, so it would be com.placelab.amila